via/static/scripts/video_player/hooks/use-side-by-side-layout.ts

```diff
@@ -60,17 +64,16 @@ export function useSideBySideLayout(container = document.body) {
   // here (versus margin) because it is part of an element's dimensions
   // (`clientWidth`) and thus easier to compute with.
   useEffect(() => {
-    const applySideBySide = sufficientSpace && sidebarOpen;
-    if (applySideBySide) {
+    if (sideBySideActive) {
       // prettier-ignore
       container.style.paddingRight = `${
         sidebarWidth - (TOOLBAR_WIDTH / 2)
       }px`;
     } else {
       container.style.paddingRight = `${TOOLBAR_WIDTH}px`;
     }
-    container.setAttribute('data-side-by-side', `${applySideBySide}`);
-  }, [container, sidebarOpen, sidebarWidth, sufficientSpace]);
+    container.setAttribute('data-side-by-side', `${sideBySideActive}`);
```

Comment:
I think this attribute may now be redundant?

Reply:
I left it in case it could be useful for debugging, but probably redundant in most of the cases. I'll remove it and we can always get it back if needed.